pkg/apis/core/validation/service_accounts_test.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
```

Comment:
Is it intentionally empty? Why we need it then?

Reply:
Just as placeholder for future tests.
I didn't find any tests dedicated for service accounts.

Reply:
Consider removing it.

Reply:
>Consider removing it.
Done